releasenotes/_deconst.json

```diff
@@ -0,0 +1,8 @@
+{
+    "contentIDBase": "https://github.com/rcbops/rpc-openstack/v13”,
+    "githubUrl": "https://github.com/rackerlabs/docs-rackspace/",
```

Comment:
My preference would be to have rpc-openstack used as the location to track release notes issues. The reason for this is that the dev team will be primarily responsible for adding notes.